internals/webpack/webpack.dev.babel.js

```diff
@@ -53,6 +53,10 @@ module.exports = require('./webpack.base.babel')({
 
   // Emit a source map for easier debugging
   devtool: 'cheap-module-eval-source-map',
+
+  performance: {
+    hints: false,
```

Comment:
Thanks for reviewing it and good catch! I don't know why I didn't think about fixing those warnings that appear during installation, because that config in `webpack.dev` only works when you run `npm run dev`, not during the generation of webpack dll file.